lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -196,17 +196,25 @@ public JourneyResponse handleRequest(JourneyRequest event, Context context) {
                 }
             }
 
-            if (userHasFaceToFaceRequest && Objects.isNull(faceToFaceVc)) {
-                var message =
-                        new StringMapMessage()
-                                .with(
-                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                        "F2F cri pending verification.");
-                LOGGER.info(message);
-                return new JourneyErrorResponse(
-                        JOURNEY_ERROR_PATH,
-                        HttpStatus.SC_INTERNAL_SERVER_ERROR,
-                        ErrorResponse.PENDING_VERIFICATION_EXCEPTION);
+            if (userHasFaceToFaceRequest) {
```

Comment:
move userHasFaceToFaceRequest & faceToFaceVc retrieve  statement (line 144 and 145) here above if block.